pulsar-client-cpp/include/pulsar/Client.h

```diff
@@ -355,6 +355,10 @@ class PULSAR_PUBLIC Client {
      */
     void shutdown();
 
+    uint64_t getNumberOfProducers();
+    uint64_t getNumberOfConsumers();
+    uint64_t getNumberOfReaders();
```

Comment:
Please add the documents to these methods like other methods so that doxygen can generate the related docs.

Reply:
ok.